### PR TITLE
TASK-1 - New text box feature

### PR DESCRIPTION
**Pull request by sqlheisenberg**
- add x y z functionality

- tested x y z

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ ovo je upustvo kako se ova aplikacija instalira
 
 Nova izmjena na aplikaciji
 New line
+New line 2
```
Comment (sqlheisenberg) [inline README.md:9]: Missing empty line at the end

**Comment by miroslav93** [comment]
review

File: README.md (hunk 1 above)
Comment (sqlheisenberg) [reply README.md:9]: This is now fixed

**Comment by ligzer** [comment]
Dobro!

**Review by mirsad-mehanovic — APPROVED** [review]

**Review by Azra1997 — APPROVED** [review]

**Pull request merged**